studiopress/genesis-custom-blocks
# Limit CircleCI to only run on affected files

#### Changes
* Only runs certain CircleCI jobs when relevant files are in the latest commit

#### Testing instructions
* Probably no local testing needed, as this only touches `.circleci/config.yml`
* Verify the last few commits:
https://github.com/studiopress/genesis-custom-blocks/pull/34/commits/75b2912cda8b5b9c7e3dbd7c14153b69b17fbeed : Should trigger PHPUnit tests, but JS tests should exit early
https://github.com/studiopress/genesis-custom-blocks/pull/34/commits/cb651d40cb7cacb4551a250215c1a4c90412f730 : Should trigger JS tests, but PHPUnit tests should exit early
Both should still have the `lint` and `e2e-tests` jobs
